LovelyCrane/LovelyCraneApp.swift

```diff
@@ -10,12 +10,16 @@ import FirebaseCore
 
 @main
 struct LovelyCraneApp: App {
+    @StateObject var viewRouter = ViewRouter()
     
     @UIApplicationDelegateAdaptor(AppDelegate.self) var delegate
     
     var body: some Scene {
         WindowGroup {
-            RootView()
+//            NavigationStack {
+                ContentView()
+                    .environmentObject(viewRouter)
+//            }
```

Comment:
```suggestion

                ContentView()
                    .environmentObject(viewRouter)
```

주석은 삭제해도 될거가탸욥 ~~~ !